cernopendata/modules/glossary/fetchers.py

```diff
@@ -0,0 +1,17 @@
+"""PID Fetchers."""
```

Comment:
Hmm, I dislike seeing `cernopendata/modules/glossary/...` structure: 

- Firstly, glossary is not really a module, it is simply one of our supported data types.  We'll have a "record" type (=now MARC, later custom JSON), a "glossary term" type, and an "article" type.  It would be nice to find a good directory structure that will cover these all later. Shouldn't we aim at a structure like "cernopendata/modules/records/glossary/...`?

- Secondly, we took pain to split "cernopendata-theme", "cernopendata-pages" into independent Python packages.  (Unlike say Zenodo that uses them in "modules" directory https://github.com/zenodo/zenodo/tree/master/zenodo.) Shouldn't we create an independent "cernopendata-records" perhaps? (This would override the above comment.)

Just musing out loud to cover future scenarios...